src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy

```diff
@@ -354,6 +359,100 @@ class SwitchSyncSpec extends BaseSpecification {
         flow && flowHelperV2.deleteFlow(flow.flowId)
     }
 
+    @Tags([VIRTUAL, LOW_PRIORITY])
+    def "Able to synchronize misconfigured default meter"() {
+        given: "An active switch with valid default rules and one misconfigured default meter"
+        def sw = topology.activeSwitches.first()
+        def broadcastCookieMeterId = MeterId.createMeterIdForDefaultRule(VERIFICATION_BROADCAST_RULE_COOKIE).getValue()
+        def meterToManipulate = northbound.getAllMeters(sw.dpId).meterEntries
+                .find{ it.meterId == broadcastCookieMeterId }
+        def newBurstSize = meterToManipulate.burstSize + 100
+        def newRate = meterToManipulate.rate + 100
+
+        lockKeeper.updateBurstSizeAndRate(sw, meterToManipulate.meterId, newBurstSize, newRate)
+        Wrappers.wait(RULES_DELETION_TIME) {
+            def validateInfo = northbound.validateSwitch(sw.dpId)
+            assert validateInfo.rules.missing.empty
+            assert validateInfo.rules.misconfigured.empty
+            assert validateInfo.meters.misconfigured.size() == 1
+            assert validateInfo.meters.misconfigured[0].actual.burstSize == newBurstSize
+            assert validateInfo.meters.misconfigured[0].expected.burstSize == meterToManipulate.burstSize
+            assert validateInfo.meters.misconfigured[0].actual.rate == newRate
+            assert validateInfo.meters.misconfigured[0].expected.rate == meterToManipulate.rate
+        }
+
+        when: "Synchronize switch"
+        northbound.synchronizeSwitch(sw.dpId, false)
+
+        then: "The misconfigured meter is fixed and moved to the 'proper' section"
+        Wrappers.wait(RULES_INSTALLATION_TIME) {
+            with(northbound.validateSwitch(sw.dpId)) {
+                it.meters.misconfigured.empty
+                it.meters.proper.find { it.meterId == broadcastCookieMeterId }.burstSize == meterToManipulate.burstSize
+                it.meters.proper.find { it.meterId == broadcastCookieMeterId }.rate == meterToManipulate.rate
+            }
+        }
+    }
+
+    @Tidy
+    @Tags([VIRTUAL, SMOKE])
+    def "Able to synchronize misconfigured flow meter"() {
+        given: "An active switch with flow on it"
+        def sw = topology.activeSwitches.first()
+        def flow = flowHelperV2.singleSwitchFlow(sw)
+        flowHelperV2.addFlow(flow)
+
+        when: "Update flow's meter"
+        def flowMeterIdToManipulate = northbound.getAllMeters(sw.dpId).meterEntries.find {
+            it.meterId > MAX_SYSTEM_RULE_METER_ID
```

Comment:
I think 1meterId >= MIN_FLOW_METER_ID` will be more correct

Reply:
fixed